tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala

```diff
@@ -150,20 +154,21 @@ class VarDictJavaEndToEnd
     val tmpVcf = f(".vcf")
 
     val vardict = new VarDictJava(
-      tumorBam          = tumorBam,
-      bed               = bed,
-      ref               = ref,
-      tumorName         = tn,
-      minimumAf         = minimumAf,
-      includeNonPf      = includeNonPf,
-      minimumQuality    = minimumQuality,
-      maximumMismatches = maximumMismatches,
-      minimumAltReads   = minimumAltReads,
-      pileupMode        = pileupMode,
-      minThreads        = minThreads,
-      maxThreads        = maxThreads
+      tumorBam            = tumorBam,
+      bed                 = bed,
+      ref                 = ref,
+      tumorName           = tn,
+      minimumAf           = minimumAf,
+      includeNonPf        = includeNonPf,
+      minimumQuality      = minimumQuality,
+      maximumMismatches   = maximumMismatches,
+      minimumAltReads     = minimumAltReads,
+      pileupMode          = pileupMode,
+      countNsInTotalDepth = countNsInTotalDepth,
+      minThreads          = minThreads,
+      maxThreads          = maxThreads
     )
-    val removeRefEqAltRows = new ShellCommand("awk", "{if ($6 != $7) print}") with PipeWithNoResources[Any,Any]
+    val removeRefEqAltRows = if (allSites) Pipes.empty[Any] else new ShellCommand("awk", "{if ($6 != $7) print}") with PipeWithNoResources[Any,Any]
```

Comment:
This is the only thing I'm not sure about.  This was here because VarDict is/was outputting VCF records where the same allele is listed in both the ref and alt columns, which I believe is invalid VCF and causes some implementations to fail to parse the results.  For sites that are all-ref it should output `.` in the alt column, not repeat the ref allele.  

When I did this, it was under the assumption that this was a bug.  If this is how it's outputting sites in pileup mode then I think we'll need to substitute something that, when ref == alt, sets alt to `.`.  What is VarDict outputting at reference sites?

Reply:
From the spec: `ALT - alternate base(s): Comma separated list of alternate non-reference alleles`

Reply:
@tfenne there will be a patch (not by me) to VarDict shortly to make it spec compliant in this case.  This PR is in anticipation of that.  I should have mentioned that, my apologies.

Reply:
@tfenne looks like the VarDict PR has been merged!

Reply:
I could find a PR in VarDict or VarDictJava that changes this behaviour, but if you're sure it's ok, go ahead.

Reply:
Found one! https://github.com/AstraZeneca-NGS/VarDict/pull/58